tools/factories.js

```diff
@@ -0,0 +1,32 @@
+import _ from 'lodash';
+import path from 'path';
+import fsp from 'fs-promise';
+import { exec } from 'child-process-promise';
+
+import components from './components';
+
+const repoRoot = path.resolve(__dirname, '../');
+const src = path.join(repoRoot, 'src');
```

Comment:
The repoRoot is now available through the tools/constants module, src would also be a good candidate to add to tools/constants if you want to.